README.md

```diff
@@ -5,33 +5,52 @@
 Lattice
 ===========
 
-A Python package that provides support for a schema-based building data model framework, currently under development as ASHRAE Standard 232P. Lattice performs data model validation, intermediate schema generation, and data file validation for YAML data models compliant with ASHRAE 232P and ASHRAE 205.
+A Python package that provides support for a schema-based building data model framework, currently under development as ASHRAE Standard 232P, where schema are described in compliant YAML source files. Lattice performs:
 
+- Data model validation: Ensures the YAML schema source files are well-formed.
+- Schema generation: Translates the YAML schema source files into equivalent JSON Schema.
+- Data file validation: Validates data files against the generated JSON Schema and additional validation requirements not supported by JSON Schema (e.g., reference checking).
+- Data model documentation: Generates web documentation of the data model from the YAML schema source files and templated markdown files (requires [Go](https://go.dev/) and [Hugo](https://gohugo.io/installation/)). This web documentation can be automatically deployed to GitHub pages.
 
-Installing the Toolkit
---------------------
+Future additions under development include:
 
-To install and use the Lattice project, you must have a supported version of Python. 
+- Generation of PDF documentation of the data model.
+- Generation of C/C++ source code for processing compliant data files.
 
-To install, simply use 
 
-`pip install .`
+Installing
+----------
 
-from the main Lattice directory. To test the schema-building process, you may add 
+Lattice is not yet available on PyPI. Currently, it is recommended to install using `pip` directly from GitHub:
 
-`poetry run pip install doit` and
-`poetry run doit`.
+`pip install "lattice @ git+https://github.com/bigladder/lattice@main"`
+
+`@main` is the cutting edge version of the package. You may alternatively use a tagged version (e.g., `@v0.1.0`) for a more stable installation.
```

Comment:
This seems anti-GitFlow - should we do a develop branch to keep main more stable?